Annotations/Anonymize.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * This file contains the Anonymize annotation class
+ *
+ * Minimal required PHP version is 5.6
+ *
+ * @category  Bundle
+ * @package   Gdpr
+ * @author    Superbrave <info@superbrave.nl>
+ * @copyright 2018 Superbrave <info@superbrave.nl>
+ * @license   https://github.com/superbrave/gdpr-bundle/blob/master/LICENSE MIT
+ * @link      https://www.superbrave.nl/
+ */
+
+namespace SuperBrave\GdprBundle\Annotation;
+
+use Doctrine\Common\Annotations\Annotation;
+use Doctrine\Common\Annotations\AnnotationException;
+
+/**
+ * Annotation to flag an entity field to be anonymized
+ * to conform with the GDPR right to be forgotten
+ *
+ * @Annotation()
+ * @Annotation\Target({"PROPERTY"})
+ */
+class Anonymize
+{
+    /**
+     * Alias of TYPE_CONCAT
+     */
+    const TYPE_STRING = 'string';
+
+    /**
+     * Anonymize a string to <columnName>-<primaryKey>
+     */
+    const TYPE_CONCAT = 'concat';
+
+    /**
+     * Anonymize a field by emptying it
+     * Alias of TYPE_FIXED with an empty value.
+     */
+    const TYPE_TRUNCATE = 'truncate';
+
+    /**
+     * Anonymize an IP field by filling in 127.0.0.1
+     * If this is a string field, INET_ATON will be used instead.
+     */
+    const TYPE_IP = 'ip';
+
+    /**
+     * Anonymize by providing a fixed value
+     * Requires the value property to be given
+     */
+    const TYPE_FIXED = 'fixed';
+
+    /**
+     * Anonymize an email to "<primaryKey>@example.com"
```

Comment:
I am wondering here, where would the domain information be grabbed from? Seems like a configuration value would be a bad idea for portability.

Reply:
We can change it so localhost is used as the host.